moonstone/analysis/statistical_test.py

```diff
@@ -38,7 +38,7 @@ def statistical_test_groups_comparison(series: pd.Series, group_series: pd.Serie
                                        output: str = 'dataframe', sym: str = True, **kwargs):
     """
     :param output: {'series', 'dataframe'}
-    :param mirror: whether generated dataframe (matrix) (or MultiIndexed series) is symetric or half-full
+    :param sym: whether generated dataframe (or MultiIndexed series) is symetric or half-full
```

Comment:
make me think that maybe you could add tests in test_statistical_test.py that handles both cases?